pos_order_mgmt/models/pos_config.py

```diff
@@ -10,26 +11,31 @@
 class PosConfig(models.Model):
     _inherit = 'pos.config'
 
+    iface_order_mgmt = fields.Boolean(
+        string='Order Management',
+        help='Allows to manage orders in the frontend',
+    )
+
     iface_reprint_done_order = fields.Boolean(
```

Comment:
Maybe rename all the field too. (With oldname) to be consistent. 
What do you think ?

Reply:
Well.. I prefer not to do that now.
I'm already working on another module that depends heavily on this one, so I don't want to break compatibility.

Personally I prefer to restric field renaming/removing to migrations. Unless it's of importance